demisto_sdk/commands/lint/resources/pylint_plugins/xsoar_level_checker.py

```diff
@@ -28,11 +33,17 @@ class XsoarChecker(BaseChecker):
     def __init__(self, linter=None):
         super(XsoarChecker, self).__init__(linter)
         self.is_script = True if os.getenv('is_script') == 'True' else False
+        self.common_args_params = ['args', 'dargs', 'arguments', 'd_args', 'data_args', 'params', 'PARAMS',
```

Comment:
are there many usages of `'params', 'PARAMS'`. if not too many - i'm ok with that creating a warning.
```suggestion
        self.common_args_params = ['args', 'dargs', 'arguments', 'd_args', 'data_args', 'params', 'PARAMS',
```